### ISSUE TITLE
Extracting image features about facial expressions

### ISSUE THREAD

**Issue by laodar** [tub-cv-group/conclugen#2]
Thanks for your excellent work. 
I want to extract features about expressions from images, which pretrained model is prefered?

**Comment by florianblume** [comment]
Hi, thank you for your interest in our work! Could you elaborate a bit further what exactly you want to do? You mean by pretrained model which one of ours?

**Comment by laodar** [comment]
> Hi, thank you for your interest in our work! Could you elaborate a bit further what exactly you want to do? You mean by pretrained model which one of ours?

I want to extract features only from image (or video) signals and use them to calculate a 'perceptual loss' that measures how close a generated image is to the ground truth image in some 'expression space'. I noticed that you trained models with many variants. Now I think model_name=contrast_only_video_audio will be prefered in my task, then how can I obtain the pretrained image feature extractor and build a 'perceptual loss' ?
